src/main/webapp/EditProfilePage.html

```diff
@@ -67,7 +67,7 @@ <h1 class="text-center">Edit Your Profile Page</h1>
             id="bio"
             name="bio"
             maxlength="250"
-            value=""
+            value="" 
```

Comment:
I didn't add a regex to the bio so that users have the option to not write anything about themselves.